vss-testclient/testclient.py

```diff
@@ -10,45 +10,89 @@
 ########################################################################
 
 import argparse, json, sys
-from cmd2 import Cmd, with_argparser, with_category
+from typing import Dict, List
 import queue, time, os
 from clientComm import VSSClientComm
-
+from pygments import highlight, lexers, formatters
```

Comment:
`pygments` is not part of the standard python library, right? Even if it is just a test dependency, it requires approval.

Reply:
@wenwenbosch Can you provide Robert with a link to the pip/github and license?

Reply:
https://github.com/pygments/pygments has BSD-2-Clause License

Reply:
And this is only part of the testclient, not part of the main server